### PR TITLE
add a preference category for hillshading, add explanatory text and link to wiki

### PR DESCRIPTION
**Pull request by ztNFny**
This ALSO addresses #13547 by replacing all android:text with android:summary in preference pages

### CODE REVIEW

File: main/res/xml/preferences_map.xml [hunk 1]
```diff
@@ -70,6 +70,15 @@
             android:summary="@string/init_summary_renderthemefolder_synctolocal"
             android:title="@string/init_renderthemefolder_synctolocal"
             app:iconSpaceReserved="false" />
+    </PreferenceCategory>
+    <PreferenceCategory
+        android:title="@string/settings_title_map_hillshading"
+        app:iconSpaceReserved="false">
+        <Preference
+            android:key="@string/pref_fakekey_info_offline_maphillshading"
+            android:text="@string/settings_info_hillshading"
```
Comment (moving-bits) [inline main/res/xml/preferences_map.xml:79]: use `android:summary` instead for the text to be displayed

**Review by moving-bits — CHANGES_REQUESTED** [review]

File: main/res/xml/preferences_map.xml (hunk 1 above)
Comment (ztNFny) [reply main/res/xml/preferences_map.xml:79]: I copied it from offline maps and themes info. Guess those are wrong as well then. Should i fix it in this PR?

Comment (moving-bits) [reply main/res/xml/preferences_map.xml:79]: I'm not sure. I've stumbled upon those as well after I'd commented on this PR, so I wonder whether there actually _should_ be an `android:text` attribute within preferences and whether we have only suppressed it somehow. Need to research this a bit first.

Comment (ztNFny) [reply main/res/xml/preferences_map.xml:79]: I opened https://github.com/cgeo/cgeo/issues/13547 for that when I noticed. Didn't have time to look into the cause yet

Comment (moving-bits) [reply main/res/xml/preferences_map.xml:79]: According to https://developer.android.com/develop/ui/views/components/settings/components-and-attributes there's no `text` attribute allowed / recognized within preferences. Maybe the old preferences lib tolerated it, but the AndroidX version does not seem to support it. (Also, during my very short research, I haven't found a single example using that attribute.)
So I guess the existing instances could be renamed to `android:summary`.

Comment (ztNFny) [reply main/res/xml/preferences_map.xml]: added a commit replacing all android:text with android:summary to this PR (3 dialogs)

**Comment by ztNFny** [comment]
This PR now also addresses https://github.com/cgeo/cgeo/issues/13547 by replacing all android:text with android:summary in preference pages

File: main/res/values/strings.xml [hunk 2]
```diff
@@ -1002,6 +1003,8 @@
...
     <string name="settings_info_themes_title">Info on Map Themes</string>
     <string name="settings_info_themes">c:geo supports custom themes for offline maps. These can be used to change the color style of the map (e.g. to have a nightview map) or to highlight certain objects like cycle paths or height lines within the map.\n\nWorking with non-zipped map themes might decrease map viewer opening performance. In this case, the theme folder may optionally be synchronized to an app-private local folder for faster access (at the cost of some storage space).</string>
+    <string name="settings_info_hillshading_title">Info on Hillshading</string>
+    <string name="settings_info_hillshading">Hillshading is a technique for creating topographic relief maps by adding a lighting effect to the map to indicate changes in elevation. Sloped will be darkened/lightened to generate a 3D-effect. The source files containing the elevation information have to be obtained manually - click this item for sources - and placed in the sources folder defined below.</string>
```
Comment (moving-bits) [inline main/res/values/strings.xml:1007]: Do you mean "sloped" or "slopes"?

**Review by moving-bits — APPROVED** [review]
LGTM, thanks for fixing the preferences' attributes as well. Just a tiny question about wording for the explanation for hillshading.

File: main/res/values/strings.xml (hunk 2 above)
Comment (ztNFny) [reply main/res/values/strings.xml]: well spotted. fixed

**Pull request merged**